src/controllers/userController.js

```diff
@@ -12,17 +12,32 @@ userController.post('/registerUser', async (req, res) => {
         if (req.body.password !== req.body.repass) {
             throw new Error('Password dismatch!');
         }
+        if (!req.body.email || req.body.email == '') {
+            throw new Error('Email is necessary!');
+        }
+        /*
+        TODO: When someone try to register as admin:
+        throw an Error 
+        or save record as regular?
+        */
+        if (req.body.role == 'admin') {
+            throw new Error('You do not have admin rights!');
+        }
+
         const userData = {
             //TODO: TEST if this check is needed
             email: req.body.email,
-            organizator: req.body.organizator ? req.body.organizator : '',
+            organizatorName: req.body.organizatorName
```

Comment:
check by design, it must be required, don't set it to empty string if it is an organizator and it mustn't apply if regular/admin. It applies for all below, check the design

Reply:
fixed